route.go

```diff
@@ -103,19 +137,26 @@ func (route *RouteSendFirstMatch) Dispatch(buf []byte) {
 	}
 }
 
+func (route *RouteConsistentHashing) Dispatch(buf []byte) {
+	conf := route.config.Load().(consistentHashingRouteConfig)
+	dest := conf.Dests()[conf.Hasher.GetDestinationIndex(buf)]
+	log.Info("route %s sending to dest %s: %s", route.key, dest.Addr, buf)
```

Comment:
needs a `// dest should handle this as quickly as it can` comment like the other routes, just to be clear.